tests/e2e/etcd_config_test.go

```diff
@@ -336,3 +337,32 @@ func TestGrpcproxyAndListenCipherSuite(t *testing.T) {
 		})
 	}
 }
+
+func TestEtcdTLSVersion(t *testing.T) {
+	skipInShortMode(t)
```

Comment:
`skipInShortMode` doesn't exist in release-3.4, how does the test passed?

Reply:
Strange yeah, when I run `TEST_OPTS="PASSES='build unit release integration_e2e functional'" make test` locally I can actually see the failure in the logs below.  I was relying on the github actions workflow though, there must be a bug with it which is worrying!

```bash
Starting 'integration_e2e' pass at Thu 16 Mar 2023 18:40:14 NZDT
Running integration and e2e tests...
# go.etcd.io/etcd/tests/e2e [go.etcd.io/etcd/tests/e2e.test]
tests/e2e/etcd_config_test.go:342:2: undefined: skipInShortMode
tests/e2e/etcd_config_test.go:362:6: too many arguments in call to spawnCmd
        have ([]string, nil)
        want ([]string)
FAIL    go.etcd.io/etcd/tests/e2e [build failed]
FAIL
```

Putting that aside - I will get these two issues fixed shortly.

Reply:
Could you raise an issue to track these two issues? thx